src/components/Tabs/TabPanel.tsx

```diff
@@ -6,16 +6,24 @@ type TabPanelProps = {
   tabName: string;
   children: React.ReactNode;
   isActive?: boolean;
+  hasBorder?: boolean;
 };
 
 /**
  * TabPanel is a compound component. TabPanel MUST BE a direct child of Tabs.
  * The `isActive` prop isn't passed in declaratively. `isActive` is passed
  * from the Tabs render from the React.Children cloneElement.
  */
-const TabPanel = ({ id, tabName, isActive, children }: TabPanelProps) => {
+const TabPanel = ({
+  id,
+  tabName,
+  isActive,
+  children,
+  hasBorder
+}: TabPanelProps) => {
   const classes = classnames('easi-tabs__tab-panel', {
-    'easi-only-print': !isActive
+    'easi-only-print': !isActive,
+    'easi-tabs__tab-panel-border': hasBorder
```

Comment:
```suggestion
    'easi-tabs__tab-panel--bordered': hasBorder
```

This should a "modifier"